platform/android/README.md

```diff
@@ -14,6 +14,21 @@ Alright. So, actually, you may be in the wrong place. From here on in, this READ
 
 [![](https://www.mapbox.com/android-sdk/images/splash.png)](https://www.mapbox.com/android-sdk/)
 
+#### Open project in Android Studio
+
+#### Using the SDK snapshot
+
+Instead of using the latest stable release of the Mapbox Android SDK, you can use the nightly build/"snapshot" or the beta version if there is one available. If you'd like to use a snapshot build,  your Android project's gradle file should
+have -SNAPSHOT appended to the SDK version number. For example `5.2.0-SNAPSHOT` or:
+
+```java
```

Comment:
```
allprojects {
    repositories {
        jcenter()
        maven { url "http://oss.sonatype.org/content/repositories/snapshots/" }
    }
}
```

You need to have above section in your root folder build.gradle to be able to resolve these dependencies ( `maven { url "http://oss.sonatype.org/content/repositories/snapshots/" }`)